modules/antismash/antismashlite/main.nf

```diff
@@ -0,0 +1,67 @@
+process ANTISMASH_ANTISMASHLITE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::antismash-lite=6.0.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/antismash-lite:6.0.1--pyhdfd78af_1' :
+        'quay.io/biocontainers/antismash-lite:6.0.1--pyhdfd78af_1' }"
+
+    containerOptions {
+        workflow.containerEngine == 'singularity' ?
+        "-B $antismash_dir:/usr/local/lib/python3.8/site-packages/antismash" :
+        workflow.containerEngine == 'docker' ?
+        "-v \$PWD/$antismash_dir:/usr/local/lib/python3.8/site-packages/antismash" :
+        ''
+        }
+
+    input:
+    tuple val(meta), path(sequence_input)
+    path(databases)
+    path(antismash_dir) // Optional input: AntiSMASH installation folder. It is not needed for using this module with conda, but required for docker/singularity (see meta.yml).
+
+    output:
+    tuple val(meta), path("${prefix}/clusterblast/*_c*.txt")                 , optional: true, emit: clusterblast_file
+    tuple val(meta), path("${prefix}/css/*.css")                             , emit: css_file
+    tuple val(meta), path("${prefix}/images")                                , emit: image_directory
+    tuple val(meta), path("${prefix}/js/*.js")                               , emit: javascript
```

Comment:
I wonder if this could be made more generic to just output the HTML here, also for the SVG stuff. I think it's unlikely you would use these downstreams (I guess clusterblash `*txt` files could maybe wanted to export on a per file basis)

Reply:
Is it possible to output several folders in a single channel? How would that pattern for the meta.yml look like for the `css/`, `images/` and `js/` folders (something like `pattern: ["css/", "images/", "js/"]`?

Reply:
I guess you could go `pattern "{css,images,js}/"`?

Reply:
I agree, it would be nice to group these together.

Reply:
Right! And in the `main.nf`, how would the resulting merged line look like?
Something like `tuple val(meta), path("${prefix}/{css,images,js}"), emit: html_accessory_files`?

Reply:
I would try that, yes. Never actually done that before but it "should" work.

Reply:
Works!